Apps/Common/src/Models/NotificationSettingsRequest.cs

```diff
@@ -31,14 +31,28 @@ public NotificationSettingsRequest()
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="NotificationSettingsRequest"/> class.
+        /// Copy constructor for a instance of the <see cref="NotificationSettingsRequest"/> class.
         /// </summary>
         /// <param name="notificationSettings">Initialize values from passed in object.</param>
         public NotificationSettingsRequest(NotificationSettingsBase notificationSettings)
             : base(notificationSettings)
         {
         }
 
+        /// <summary>
```

Comment:
This seems to go against the norm and creates a constructor when we can use an object initializer.

Reply:
We can, but to of the variables can be derived so I believe the constructor is better in this case

Reply:
When I saw the usage, I like it better as well.